rts/lra-examples/cdi-embedded/README.md

```diff
@@ -0,0 +1,21 @@
+# MicroProfile LRA Participant Example with Embedded Coordinator
+
+This example is a copy of the [`cdi-participant`](#../cdi-participant/README.md) 
+example but instead of using an external LRA coordinator one is embedded
```

Comment:
typo

Reply:
There are no typos here.

Reply:
I thought here `this` was missing.
"example but instead of using an external LRA coordinator `,this` one is embedded"

Reply:
`one is embedded` is referring to the subject `external LRA coordinator`.

Reply:
I will simplify the sentence construction.

Reply:
Okay Thanks